styles.css

```diff
@@ -19,10 +19,36 @@ body {
 .logo,
 .menu {
   display: inline-block;
+  padding: 40px 0 40px 0;
 }
 
 .logo {
-  padding: 40px 0;
+  width: 190px;
+  height: 35px;
+  padding-left: 44px;
+  font: bold 24px/31px sp_serif, arial, helvetica, sans-serif;
+  position: relative;
+}
+
+.logo__circle {
+  left: 0;
+  top: 50%;
+  width: 28px;
+  height: 28px;
+  position: absolute;
+  transform: translateY(-50%);
+  border-radius: 50%;
+  opacity: .7;
+}
+
+.logo__circle_color_dark-blue {
+  background: #0056EC;
+}
+
+.logo__circle_color_blue {
+  left: 8px;
+  top: 56%;
+  background: #C8DBFB;
```

Comment:
Codacy found an issue: [Expected "#C8DBFB" to be "#c8dbfb" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/-Kostikova-Irina/pullRequest?prid=6241956)